modules/onnx/1.17.0/MODULE.bazel

```diff
@@ -0,0 +1,6 @@
+module(name = "onnx", version = "1.17.0")
+
+bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(name = "rules_python", version = "0.31.0")
+bazel_dep(name = "rules_proto", version = "6.0.0-rc2")
```

Comment:
I believe we should use `proto_library` from protobuf: https://github.com/protocolbuffers/protobuf/blob/b4704d1933500ef35569405bfc8fe23fe64c94ad/bazel/proto_library.bzl#L14